onboarding/source/controller.c

```diff
@@ -70,15 +76,15 @@ static void controllerTask(void * pvParameters) {
     if (lightServiceStatus == 0) {
         /* USER CODE BEGIN */
         // Deal with error when initializing light service task and/or queue
-
+        sciPrintText(scilinREG, "Error", 6);
```

Comment:
Add a `\n` at the end of the error message.

Reply:
Done